demo/talkingdata-adtracking-fraud-detection/README.md

```diff
@@ -0,0 +1,78 @@
+# TalkingData AdTracking Fraud Detection
+
+We demonstrate how to use [OpenMLDB](https://github.com/4paradigm/OpenMLDB) together with other opensource software to develop a complete machine learning application for TalkingData AdTracking Fraud Detection (read more about this application on [Kaggle](https://www.kaggle.com/c/talkingdata-adtracking-fraud-detection/overview)).
+
+## Prepare
+
+### OpenMLDB
+#### Run in Docker
+
+We recommend you to use docker to run the demo. OpenMLDB and dependencies have been installed.
+
+**Start docker**
+```
+docker run -it 4pdosc/openmldb:0.5.0 bash
+```
+In the container, 
+```
+cd /work/talkingdata
+```
+
+#### Run locally
+
+Download OpenMLDB server pkg, version >= 0.5.0 .
+
+Install all dependencies:
+```
+pip install pandas xgboost==1.4.2 tornado "openmldb>=0.5.0"
```

Comment:
these libs have been installed when docker building. it is need to install again?

Reply:
It's for ' Run locally'